specification/trace/semantic_conventions/deployment.md

```diff
@@ -0,0 +1,109 @@
+# Semantic conventions for Application Deployment
+
+**Status**: [Experimental](../../document-status.md)
+
+This document defines semantic conventions for recording application delivery as span events.
+To record an evaluation outside a transaction context, consider
+[recording it as a log record](../../logs/semantic_conventions/deployment.md).
+
+<!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
+
+<!-- toc -->
+
+- [Overview](#overview)
+- [Definitions](#definitions)
+  * [Application](#application)
+  * [Workload](#workload)
+  * [Task](#task)
+
+<!-- tocstop -->
+
+## Overview
+
+Deployments of cloud-native applications are automated and done at a fast phase.
+In most cases, deployments are complex, require multiple steps, could spawn across multiple environments,
+and are delivered mostly via either Operators or GitOps practices and tools.
+Hence, the delivery process is modeled as an event stream of OpenTelemetry [`Event`s](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#add-events).
+This document defines semantic conventions to collect telemetry signals about events that occur during the
+deployment of an application.
+
+## Definitions
+
+### Application
+
+Each stage of the lifecycle of a cloud-native application deployment SHOULD be recorded as an Event on the span during
```

Comment:
Should these events have any relation with the [Continuous Deployment Events](https://github.com/cdevents/spec/blob/v0.3.0/continuous-deployment.md) of [CDEvents](https://cdevents.dev/)?